HIP/hip-415.md

```diff
@@ -0,0 +1,262 @@
+---
+hip: 415
+title: Introduction Of Blocks
+author: Daniel Ivanov<daniel-k-ivanov95@gmail.tech>, Ivan Kavaldzhiev<ivan.kavaldzhiev@limechain.tech>
+working-group: Danno Ferrin<@shemnon>, Richard Bair<@rbair23>, Steven Sheehy<steven.sheehy@hedera.com>, Mitchell Martin<mitch@swirlds.com>
+type: Standards Track
+category: Core
+needs-council-approval: Yes
+status: Last Call
+created: 2022-03-28
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/434
+last-call-date-time: 2022-05-03
+requires: HIP-435
```

Comment:
When 435 gets into master, link to it here [HIP-435](https://hips.hedera.com/hip/hip-435)